modules/machineset-yaml-aws.adoc

```diff
@@ -125,13 +125,13 @@ $ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 ifndef::infra[]
 <2> Specify the infrastructure ID, node label, and zone.
 <3> Specify the node label to add.
-<4> Specify a valid {op-system-first} AMI for your AWS zone for your {product-title} nodes.
+<4> Specify a valid {op-system-first} AMI for your AWS zone for your {product-title} nodes. If you want to use an AWS Marketplace image, you must complete the {product-title} subscription from the link:https://aws.amazon.com/marketplace/fulfillment?productId=59ead7de-2540-4653-a8b0-fa7926d5c845[AWS Marketplace] to obtain an AMI id for your region.
```

Comment:
```suggestion
<4> Specify a valid {op-system-first} AMI for your AWS zone for your {product-title} nodes. If you want to use an AWS Marketplace image, you must complete the {product-title} subscription from the link:https://aws.amazon.com/marketplace/fulfillment?productId=59ead7de-2540-4653-a8b0-fa7926d5c845[AWS Marketplace] to obtain an AMI ID for your region.
```

Reply:
I forgot to add the section in the ISG for this: https://www.ibm.com/docs/en/ibm-style?topic=word-usage